### ISSUE TITLE
2020 Smith County Imagery 

### ISSUE THREAD

**Issue by csidenblad** [TNRIS/incoming-data#47]
Please reassign issues when your work is complete.

Data

- [x]  Enter dataset on StratMap Incoming data xlsx sheet @gmullins
- [x] Route Data Shipment (copy data to TNRIS Projects share) @csidenblad
- [x]  Data Processing (renaming file, data organization) @mitchellryant @csidenblad
~~- [ ]  Creation of derivative products [compressed images] @csidenblad~~
- [x]  Quality Assurance @csidenblad 

DataHub

- [x]  Create collection @mitchellryant
- [x]  Complete console collection form including metadata and description @csidenblad 
- [x]  Create Images, place on Website Dir @csidenblad 
- [x]  Crop & Upload Images @csidenblad 
- [x]  Upload supplemental info (breaklines, reports, index) @gmullins @csidenblad 
If zip file is too large (more than 75MB) please let @mitchellryant know and the file can be loaded in another way.
- [x]  Review Description @pwblanton
- [x]  Add resources @mitchellryant

Download\Prod

- [x]  Move data to production shares @mitchellryant
- [x]  Make collection public @mitchellryant
- [ ]  WMS Link @mitchellryant
~~- [ ]  Inform TNRIS staff of data availability @mitchellryant~~

**Comment by mitchellryant** [comment]
@gmullins is there metadata?

**Comment by mitchellryant** [comment]
@gmullins smith20 or smith19?

all files have been copied from OneDrive to Gorgoroth.

**Comment by gmullins** [comment]
@mitchellryant   smith20
John from SmithCAD was to upload metadata to my OneDrive. Haven't heard if he's done yet.
Thanks!

**Comment by mitchellryant** [comment]
@gmullins the metadata has been copied to the network.

**Comment by csidenblad** [comment]
@mitchellryant photos have been cropped and uploaded. Saved them in M:\SM\Graphics\Imagery\Smith_2020 , if this is not the correct location. If not, let me know and i'll move them.

**Comment by LaurenKirk** [comment]
@pwblanton 

The Smith County GIS Consortium 2020 6-inch Digital aerial imagery was acquired, processed to orthophotography and delivered to display approximately 1,104 square miles of the greater countywide area. The Smith County area orthoimagery delivery includes 1,104 Quarter-Quarter Quads (DO4Qs) uncompressed GeoTIFF tiles and 1,104 DO4Q compressed JPEG2000 tiles and 1,104 DO4Q ECW tiles, countywide JPEG2000 and ECW mosaics along with supporting files. The aerial imagery for the entire countywide project area was collected and processed exclusively by Surdex Corporation. The Statement of Work is issued by the Smith County GIS Consortium to acquire leaf-off high resolution 6-inch orthoimagery and associated products from an airborne digital camera system. The Smith County GIS Consortium consists of the Smith County 911 Communications District, City of Tyler, Smith County Appraisal District, and Smith County. This 6-inch resolution leaf-off aerial imagery was acquired in three (3) missions all flown on December 11, 2019. This product was produced to be used by a wide variety of local entities for city engineering, city planning, 9-1-1 database updates, general mapping, appraisal district valuations, infrastructure mapping and other applications. To begin the project, existing and newly surveyed ground control survey points were utilized to control and check the aerial imagery. Surdex utilizes the IGI Plan flight planning software for the ADS100 sensor acquisition planning. The IGI software utilizes an elevation model to rigorously check for adequate forward-lap and side-lap coverage of the area of interest. The ADS100 is a pushbroom sensor consisting of linear arrays oriented across the flight path, imaging one row of pixels by 20,000 pixels as the aircraft moves along. The resolution of each array is the same, thus requiring no "pan-sharpening" to create color, color infrared (CIR), or 4-band (R,G,B,N) images. The imagery was captured utilizing two (2) individual aircraft and ADS100 sensors (Conquest 441 – tail number N440EH with ADS100 10552 and Conquest 441 – tail number N441EH with ADS100 10515) at a flying height of ~6,100' above ground level. All products are delivered in NAD 1983 (2011) State Plane Texas North Central FIPS 4202 Feet.

**Comment by pwblanton** [comment]
Chelsea, I reviewed the Smith County Imagery write-up and it looks good. Thanks!

**Comment by mitchellryant** [comment]
@csidenblad I added some breaks and removed the tail number. api has been updated

The Smith County GIS Consortium 2020 6-inch Digital aerial imagery was acquired, processed to orthophotography and delivered to display approximately 1,104 square miles of the greater countywide area. The Smith County area orthoimagery delivery includes 1,104 Quarter-Quarter Quads (DO4Qs) uncompressed GeoTIFF tiles and 1,104 DO4Q compressed JPEG2000 tiles and 1,104 DO4Q ECW tiles, countywide JPEG2000 and ECW mosaics along with supporting files. The aerial imagery for the entire countywide project area was collected and processed exclusively by Surdex Corporation. The Statement of Work is issued by the Smith County GIS Consortium to acquire leaf-off high resolution 6-inch orthoimagery and associated products from an airborne digital camera system. 

The Smith County GIS Consortium consists of the Smith County 911 Communications District, City of Tyler, Smith County Appraisal District, and Smith County. This 6-inch resolution leaf-off aerial imagery was acquired in three missions all flown on December 11, 2019. This product was produced to be used by a wide variety of local entities for city engineering, city planning, 9-1-1 database updates, general mapping, appraisal district valuations, infrastructure mapping and other applications. To begin the project, existing and newly surveyed ground control survey points were utilized to control and check the aerial imagery. Surdex utilizes the IGI Plan flight planning software for the ADS100 sensor acquisition planning. The IGI software utilizes an elevation model to rigorously check for adequate forward-lap and side-lap coverage of the area of interest. The ADS100 is a pushbroom sensor consisting of linear arrays oriented across the flight path, imaging one row of pixels by 20,000 pixels as the aircraft moves along. The resolution of each array is the same, thus requiring no "pan-sharpening" to create color, color infrared (CIR), or 4-band (R,G,B,N) images. 

The imagery was captured utilizing two individual aircraft and ADS100 sensors at a flying height of ~6,100' above ground level. 

All products are delivered in NAD 1983 (2011) State Plane Texas North Central FIPS 4202 Feet.
